### PR TITLE
Adding the VS15.0 image

### PR DESCRIPTION
**Pull request by tannergooding**
FYI. @mmitche, @jasonmalinowski, @dotnet/roslyn-infrastructure

### CODE REVIEW

File: jobs/generation/Utilities.groovy [hunk 1]
```diff
@@ -243,6 +245,8 @@ class Utilities {
                                 'latest-or-auto-dev15-rc2-internal':'win2016-20170105-internal',
                                 // For internal runs - Win2016 + VS15.RC4
                                 'latest-or-auto-dev15-internal':'win2016-20170214-internal',
+                                // For internal runs - Win2016 + VS15.0
+                                'latest-or-auto-vs15-0-interanl':'win2016-20170307-internal',
```
Comment (mmitche) [inline jobs/generation/Utilities.groovy]: Typo (interanl)

Comment (tannergooding) [inline jobs/generation/Utilities.groovy]: Fixed.

File: management/nodes/startupScripts/win2016-20170307-internal.ps1 [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+echo ('Virtual Machine Name: {0}' -f $vmName) | Out-File -Append $machineSetupLog
+$secret = $args[2]
+echo ('Secret: {0}' -f $secret) | Out-File -Append $machineSetupLog
+
+$winLogonRegistryPath = 'hklm:\software\microsoft\windows nt\currentversion\winlogon\'
+$jenkinsDirectory = 'C:\Jenkins'
+$jenkinsLaunchCmd = Join-Path -Path $jenkinsDirectory -ChildPath 'launch.cmd'
+$jenkinsStartupScript = Join-Path -Path $jenkinsDirectory -ChildPath 'jenkins-windows-startup.ps1'
+$jenkinsLogFile = Join-Path -Path $jenkinsDirectory -ChildPath 'log.txt'
+# The image needs to have LUA off to run properly. If you want the VM to have administrative permissions for
```
Comment (jasonmalinowski) [inline management/nodes/startupScripts/win2016-20170307-internal.ps1]: This comment and the `$enableLUA = ...` line seems to be at odds with each other.

File: management/nodes/startupScripts/win2016-20170307.ps1 [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+echo ('Virtual Machine Name: {0}' -f $vmName) | Out-File -Append $machineSetupLog
+$secret = $args[2]
+echo ('Secret: {0}' -f $secret) | Out-File -Append $machineSetupLog
+
+$winLogonRegistryPath = 'hklm:\software\microsoft\windows nt\currentversion\winlogon\'
+$jenkinsDirectory = 'C:\Jenkins'
+$jenkinsLaunchCmd = Join-Path -Path $jenkinsDirectory -ChildPath 'launch.cmd'
+$jenkinsStartupScript = Join-Path -Path $jenkinsDirectory -ChildPath 'jenkins-windows-startup.ps1'
+$jenkinsLogFile = Join-Path -Path $jenkinsDirectory -ChildPath 'log.txt'
+# The image needs to have LUA off to run properly. If you want the VM to have administrative permissions for
```
Comment (jasonmalinowski) [inline management/nodes/startupScripts/win2016-20170307.ps1]: Ditto: comment is backwards from what it is doing.

**Comment by jasonmalinowski** [comment]
@tannergooding What's the difference between the two scripts?

**Comment by tannergooding** [comment]
@jasonmalinowski, nothing. They are identical and are identical to the scripts that we had before. I'm just following the existing convention of copying and renaming said script for each new image.

**Pull request merged**